environment-pipelines/buildspec-plan.yml

```diff
@@ -42,5 +42,4 @@ phases:
         fi
 artifacts:
   files:
-    - terraform/environments/${ENVIRONMENT}/plan.tfplan
-    - copilot/environments/${ENVIRONMENT}/manifest.yml
+    - "**/*"
```

Comment:
Would this include all the files in the source directory?

Might the following be a bit leaner, but include all the things we are interested in?
```suggestion
    - "terraform/environments/**/*"
    - "copilot/environments/**/*"
```

Reply:
We still need everything from the top level for the tooling that the build stage creates. We'd have to then merge the two input sources together which would be a faff. This is cleaner.

Reply:
Yes, but it helps simplify the stage input/outputs because it also includes all of the build tools from the initial build stage